src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -28,7 +28,7 @@ class ResourceType(Enum):  # pylint: disable=too-few-public-methods
 
     MGMT_STORAGE = ('azure.mgmt.storage',
                     'StorageManagementClient')
-    MGMT_COMPUTE = ('azure.mgmt.compute.compute',
+    MGMT_COMPUTE = ('azure.mgmt.compute',
```

Comment:
With the new SDK, is `azure.mgmt.compute` still correct?

Reply:
yes